### DISCUSSION TITLE
Language workers won't load

### DISCUSSION THREAD

**Discussion by V3RON** [suren-atoyan/monaco-react#521] [Q&A]
Hi!

First of all, thank for your work!

I have to include Monaco in the Vite project I'm working on. Because of reasons it must be loaded from the NPM repository. I followed the instructions listed in the documentation (https://github.com/suren-atoyan/monaco-react#use-monaco-editor-as-an-npm-package) and it partially works. I can see the editor, syntax highlighting is working fine, but I cannot say the same about error reporting. It's basically not reporting anything.

I've tried to debug and fix it myself for hours, but I didn't make any progress. The `getWorker` function is called only once to run this basic 'editor service', but it's not called again for any other language. There are no errors in the console and I'm pretty much stuck.

Am I missing something? I'm not sure if it's the `monaco-react` wrapper problem or the `monaco` itself. 
Any advice?

**Comment by vladislav-shkodin-reflek** [comment]
Experience the same problem
